docs/cristin-proxy-swagger.yaml

```diff
@@ -3090,6 +3090,40 @@ components:
                 description: Two letter country code (ISO 3166-1 alpha-2)
               label:
                 $ref: '#/components/schemas/Languages'
+        awards:
+          type: array
+          items:
+            type: object
+            properties:
+              title:
+                type: string
+                description: Title of the award
+              year:
+                type: integer
+                description: Year of the award
+              type:
+                type: object
+                properties:
+                  type:
+                    enum:
+                      - FORSKFORMIDL
+                      - FORSKARBEID
+                    description: The code for this type of activity. Can be FORSKFORMIDL (Dissemination of research results) or FORSKARBEID (Research activity)
+                  label:
+                    $ref: '#/components/schemas/Languages'
+              distribution:
+                type: object
+                properties:
+                  type:
+                    enum:
+                      - INTERNAT
+                      - NATIONAL
+                      - INTERN
+                    description: The code for the scope of award. Can be INTERNAT (International), NATIONAL (National) or INTERN (Own institution)
```

Comment:
Same here @brinxmat . Keep enums or call them something else?

Reply:
International
National
Internal

We don't need to confuse people just to save a few bytes

Reply:
Done